api/v1/openapi.yaml

```diff
@@ -831,6 +831,9 @@ definitions:
       k8s-namespace:
         description: Kubernetes namespace name
         type: string
+      map-id:
```

Comment:
Please call this `policy-map-id`

Reply:
Changed it into data-path-map-id since this denotes the tail call map for the pod that cilium is managing where it attaches bpf_lxc onto, so only policy-map-id would be too confusing as well.